test/teos/unit/cli/test_teos_cli.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pytest
+import sys
+from unittest.mock import MagicMock
+
+from teos.cli.teos_cli import show_usage, CLI
+
+
+@pytest.fixture
+def cli():
+    cli = CLI(".teos-cli-test", {})
+    yield cli
+    os.rmdir(".teos-cli-test")
+
+
+def test_show_usage_does_not_throw():
+    # If any of the Cli commands' docstring has a wrong format and cannot be parsed, this will raise an error
+    show_usage()
+
+
+def test_cli_init_does_not_throw():
+    try:
+        cli = CLI(".teos-cli-test", {})
```

Comment:
No need to store it

Reply:
Removed assignment in 5267444f9044725c10fb19b89962b33f22d454e9.